ocp_resources/storage_profile.py

```diff
@@ -18,7 +18,7 @@ def claim_property_sets(self):
         return self.instance.status.get("claimPropertySets")
 
     def first_claim_property_set_access_modes(self):
-        return self.claim_property_sets[0].get("volumeMode") if self.claim_property_sets else None
+        return self.claim_property_sets[0].get("accessModes") if self.claim_property_sets else None
```

Comment:
If we want to get the **first** access mode, shouldn't it be
`claim_property_sets[0].get("accessModes")[0]`
?